lib/Image/Adapter/Imagick.php

```diff
@@ -284,7 +296,12 @@ public function save($path, $format = null, $quality = null)
             $i->setImageFormat($format);
             $success = File::put($path, $i->getImageBlob());
         } else {
-            $success = $i->writeImage($format . ':' . $path);
+
+            if ($this->isPreserveAnimation() && $i->getNumberImages() > 1) {
+                $success = $i->writeImages($format . ':' . $path, true);
```

Comment:
The target format needs to be `GIF` here right? 
We also have to think about for what target formats the `preserveAnimation` option should work, I guess only "Auto", "ORIGINAL" and GIF makes actually sense. But this needs to be handled somewhere properly, because currently it would be possible to specify `JPEG` as the target format 😊

Reply:
thx, done